src/pymmcore_plus/mda/_engine.py

```diff
@@ -32,6 +58,10 @@ def setup_sequence(self, sequence: MDASequence) -> None:
 
         self._mmc = self._mmc or CMMCorePlus.instance()
 
+        # switch off autofocus device if it is on
+        with contextlib.suppress(RuntimeError):
+            self._mmc.setProperty(self._mmc.getAutoFocusDevice(), "State", "Off")
```

Comment:
why not:

`self._mmc.setContinuousFocus(False)`

Reply:
@ianhi you meant `self._mmc.enableContinuousFocus(False)`? Sure, it should work as well!